src/components/ErrorBoundary/index.test.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import renderer from 'react-test-renderer';
```

Comment:
you could simply choose one of the tool - react-test-renderer or enzyme for all tests. it 's not necessary to use both

Reply:
Will leave for example.